.github/workflows/publish.yml

```diff
@@ -0,0 +1,105 @@
+name: Publish Build Artifacts
```

Comment:
Can you comment on why/how this workflow applies to us?

Reply:
It doesn't, good catch. I'll look at disabling it. We could delete the file all-together but I want to avoid merge conflicts in the future.